src/coreComponents/constitutive/fluid/CellBasedFluxSlurryFluidAccessors.hpp

```diff
@@ -0,0 +1,57 @@
+/*	
+ * ------------------------------------------------------------------------------------------------------------	
+ * SPDX-License-Identifier: LGPL-2.1-only	
+ *	
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC	
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University	
+ * Copyright (c) 2018-2020 Total, S.A	
+ * Copyright (c) 2020-     GEOSX Contributors	
+ * All right reserved	
+ *	
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.	
+ * ------------------------------------------------------------------------------------------------------------	
+ */
+
+#ifndef GEOSX_CELLBASEDFLUXSLURRYFLUIDACCESSORS_HPP
+#define GEOSX_CELLBASEDFLUXSLURRYFLUIDACCESSORS_HPP
+
+#include "SlurryFluidExtrinsicData.hpp"
+
+#include "physicsSolvers/fluidFlow/StencilAccessors.hpp"
+
+namespace geosx
+{
+
+namespace ProppantTransportKernels
+{
+
+class CellBasedFluxSlurryFluidAccessorsImpl
+{
+public:
+  CellBasedFluxSlurryFluidAccessorsImpl( ElementRegionManager const & elemManager,
+                                         string const & solverName,
+                                         arrayView1d< string const > const & regionNames,
+                                         arrayView1d< string const > const & materialNames )
+    : m_impl( elemManager, solverName, regionNames, materialNames )
```

Comment:
Note that in this case, the 
```c++
  StencilAccessors( ElementRegionManager const & elemManager, string const & solverName )
```
constructor is not exposed, and cannot be used (by mistake, _e.g._).